patch/llpcFragColorExport.cpp

```diff
@@ -1376,13 +1374,15 @@ Value* FragColorExport::Run(
                     comps[i] = pUndefFloat;
                 }
 
-                std::vector<Attribute::AttrKind> attribs;
-                attribs.push_back(Attribute::ReadNone);
+                Attribute::AttrKind attribs[] = {
+                    Attribute::ReadNone
+                };
```

Comment:
Since the array is a singleton, this can presumably be simplified further?

Reply:
You're right, I could just write `Attribute::AttrKind attribs = Attribute::ReadNone;`. I am slightly prejudiced against this simply because I didn't know that you can implicitly construct an ArrayRef from a single element.

Reply:
I think you can place this in the function directly { Attribute::ReadNone } without defining this.

Reply:
I suggest this because placing the attribs in EmitCall is more clear. We do not have to check what the value of attribs is. And in most cases, we only have one attributes in the list. It is up to your decision. I also consider we can use "NoAttrib" directly for those simple ALU functions.

Reply:
I will leave it like it is, but I don't mind if someone else wants to change it.